channeldb/kvdb/etcd/stm.go

```diff
@@ -247,19 +247,19 @@ loop:
 
 		default:
 		}
-
-		// Apply the transaction closure and abort the STM if there was an
-		// application error.
+		// Apply the transaction closure and abort the STM if there was
+		// an application error.
 		if err = apply(s); err != nil {
-			break loop
+			return err
```

Comment:
So the change here is we don't want to report commit stats if there's an _application level_ error in the transaction closure?

Reply:
no, this wasn't intentional, fixed